configure.ac

```diff
@@ -278,6 +279,8 @@ fi
 OPAL_SET_LIB_PREFIX([])
 m4_ifdef([project_orte],
           [ORTE_SET_LIB_PREFIX([])])
+m4_ifdef([project_ompi],
+          [OMPI_SET_LIB_PREFIX([ompi_lib_prefix])])
```

Comment:
Per another email thread, those were added for a different external project (ORCM, I believe). Where they wanted to prefix `libopen-pal.so` to something like `liborcm_open-pal.so`. I just piggybacked on that mechanism to achieve this patch.

If there is no reason to have the `OPAL_LIB_PREFIX` and `ORTE_LIB_PREFIX` then we can probably remove them. But if there is a chance that other projects might want to do such renaming then we might as well leave them in. They are not hurting anything.